src/parser/flags.ts

```diff
@@ -42,6 +43,20 @@ export const integer = build({
   },
 })
 
+/**
+ * Accepts a directory path.  Validates that the directory exists and is a directory.
+ */
+export const existingDirectory = build<string>({
```

Comment:
What do you think about changing these to `directory` and `file` instead? And then you can specify `exists: true` in the configuration if you need the directory or file to exist

Reply:
what would they do if they weren't `exists: true` ?

Also, none of the other other flags have a configuration beyond the OptionFlag stuff.  We *could* make them more like what we're doing in the sf-core-plugins where there's a need to pass in extra parameters (wrapping flag.build and returning the result of calling it) but that's not how the other oclif flags formed from OptionFlag work.

Reply:
If `exists: false` it would just validate that you're giving it a valid dir/file path.